apps/tlv-app/grails-app/assets/javascripts/menus/search.js

```diff
@@ -264,6 +266,113 @@ function checkAllSensors( labelElement ) {
 
 }
 
+function coverageSearch() {
+	if ( !tlv.reachbackUrl ) {
```

Comment:
What would cause this value ever be false?  if checking for undefined use typeof !== undefined.  Otherwise this will ONLY catch false values and let through undefined (and anything else).

Reply:
If it's not there, it would be null, not undefined. It comes from the config and the value it set every time, it's just that if it can't find it in the config, then it sets it to null.